### ISSUE TITLE
Got error when using number as value

### ISSUE THREAD

**Issue by KienPM** [tbleckert/react-select-search#66]
I got this error when using number as value
```
Uncaught TypeError: this.state.value.slice is not a function
    at SelectSearch.chooseOption (index.js:509)
    at onClick (index.js:670)
```
I think this line cause the error
```
var currentValue = this.state.value.slice();
```
Hope that you can update it! Thank you so much!

**Comment by matheus-garcia** [comment]
yeah, had to use toString() to handle that.

**Comment by tbleckert** [comment]
Using anything other than strings should result in a warning in dev mode because of https://github.com/tbleckert/react-select-search/blob/master/src/types.js#L5 .

But yeah, I agree. We should definitely handle both strings and numbers. I'll make a patch during the week if no one beats me to it.

**Comment by tbleckert** [comment]
Fixed in 2.0.5

**Comment by vvo** [comment]
Hey there, just confirming that I tested the new fix and it's working well, thanks a lot!
